src/views/main/SignIn.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import GoogleSignIn from "react-google-signin";
+
+class SignIn extends Component {
+
+    
+    onSignIn(userProfile, accessToken) {
+        console.log(userProfile)
+    }
+    
+    signOut() {
+        this.googleAuth.signOut();
+    }
+
+    render() {
+        return (
+          <div>
+                <GoogleSignIn clientId="680107599193-4egd5q12p1ijqor26eh0ucj369fjf037"
+                      ref={g => this.googleAuth = g}
+                      onSuccess={this.onSignIn.bind(this)}/> 
+            
+                <button onClick={this.signOut.bind(this)}> Sign Out </button>  
```

Comment:
change {this.signOut.bind(this)} -> {this.signOut}  and add constructor
please see profile.jsx